src/main/java/io/confluent/connect/hdfs/HdfsSinkConnecorConstants.java

```diff
@@ -17,4 +17,5 @@
 public class HdfsSinkConnecorConstants {
   public static final String COMMMITTED_FILENAME_SEPARATOR = "+";
   public static final String COMMMITTED_FILENAME_SEPARATOR_REGEX = "[\\.|\\+]";
+  public static final String TEMPFILE_DIRECTORY = "/tmp/";
```

Comment:
Is this going to be acceptable to everyone as a hard-coded setting? Or does this need to be configurable?